backend/src/middleware/passwordMiddleware.js

```diff
@@ -11,10 +11,11 @@ export default {
     }
   },
   Query: async (resolve, root, args, context, info) => {
-    const result = await resolve(root, args, context, info)
-    return walkRecursive(result, ['password'], () => {
+    let result = await resolve(root, args, context, info)
+    result = walkRecursive(result, ['password', 'privatKey'], () => {
```

Comment:
```suggestion
    result = walkRecursive(result, ['password', 'privateKey'], () => {
```
![Scared](https://media.giphy.com/media/9kKlRx7qpUGze/giphy.gif)